docs/UserGuide.md

```diff
@@ -27,7 +27,7 @@ Each aspect of MediBase3 is split into different sections, which are accessible
 
 **For experienced MediBase3 users:**
 1. If you need a quick refresher on how to use a specific feature in MediBase3, you can navigate to the [Features](#features) section.
-1. You may also skip to the [Command Summary](#command-summary) section for a quick overview of all the commands available in MediBase3 and their formats.
+1. You may also skip to the [Command Summary](#command-summary) section for a quick overview of all the commands available in MediBase3 and his/her formats.
```

Comment:
Would need to edit this from `his/her` formats back to `their`

Reply:
It should be `their` format